Menus/RadialMenu/RadialMenu.cs

```diff
@@ -83,6 +83,8 @@ public bool visible
 
 		public Node? node { get; set; }
 
+		public float hideDistance {get { return Mathf.Infinity; } }
```

Comment:
The idea is that the menu should always hide on any UI hover, rather than the distance-based check on mainmenu. Since the radial menu never extends beyond the rayorigin, the distance-based check doesn't apply.

To be honest, I don't notice the radial menu showing/hiding since it's so transparent. I wasn't sure if "never hide" or "hide on every hover regardless of distance" was the right default.